cmd/system-shutdown/system-shutdown.c

```diff
@@ -27,20 +27,25 @@
 #include <sys/reboot.h>		// reboot, RB_*
 #include <fcntl.h>		// open
 #include <errno.h>		// errno, sys_errlist
+#include <limits.h>             // LINE_MAX
+#include <linux/reboot.h>       // LINUX_REBOOT_MAGIC*
+#include <sys/syscall.h>        // SYS_reboot
 
 #include "system-shutdown-utils.h"
 #include "../libsnap-confine-private/string-utils.h"
 
 int main(int argc, char *argv[])
 {
+	char reboot_arg[LINE_MAX] = { 0 };
```

Comment:
I think we can reasonably say the size is something smaller, you don't need all of that space.